### DISCUSSION TITLE
What are the Logo Usage Guidelines if any?

### DISCUSSION THREAD

**Discussion by DavesCodeMusings** [micropython/micropython#9847] [General]
I'd like to use the MicroPython logo on a Visual Studio Code extension I created for mpremote (mostly so when people find it in the VS Code Marketplace, they know it's for MicroPython.) Are there usage guidelines for the logo that I should be aware of or can I just download a PNG and slap it on there? If usage is unrestricted, is there a preference between the snake sitting on a chip logo vs. the stylized M logo?

**Comment by sosi-deadeye** [comment]
https://micropython.org/termsandconditions/

>  8. Copyright and Intellectual Property
> 
> 8.1. The Website, the Purchase Services and all of the related products of George Robotics Ltd are subject to copyright. The material on the Website is protected by copyright under the laws of the United Kingdom and through international treaties. Unless otherwise indicated, all rights (including copyright) in the site content and compilation of the website (including text, graphics, logos, button icons, video images, audio clips and software) (the "Content") are owned or controlled for these purposes, and are reserved by George Robotics Ltd or its contributors.

I'm not a lawyer. I guess if you put the Logo on your page with a link to micropython.org, it should be ok, if you're using the unmodified Code from the master.

The Code is free, but the Logo not.

**Reply by DavesCodeMusings** [reply]
Thank you for the link. I'm not a lawyer either, but I tend to agree with your assessment after reading the Terms and Conditions. I would boil it down to "You can use the logo to promote our stuff, not your stuff." Kind of like the [Raspberry Pi logo conditions](https://www.raspberrypi.com/trademark-rules/).

For anyone else in my situation, there is a lovely SVG snake icon created by Colton Wiscombe over at https://materialdesignicons.com/. Just search for "snake". You can fiddle with the stroke and fill colors if you don't want monochrome gray.
